### PR TITLE
reorganize the repo

### PR DESCRIPTION
**Pull request by wjcunningham7**
- [ ] I have added the tests to cover my changes.
- [ ] I have updated the documentation and CHANGELOG accordingly.
- [ ] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by wjcunningham7** [comment]
still to-do:
- update the actions
- test docker containers

**Comment by wjcunningham7** [comment]
when testing the new structure, ensure that all remnants of a previous installation are gone, and old processes are killed.

build is tested for now using
```
python setup.py sdist
pip install dist/*.tar.gz
```

**Comment by wjcunningham7** [comment]
since this is an unavoidably massive PR, I recommend we collaborate to get the actions / unit tests working in this branch before merging.  almost all of the changes are related to file imports / naming so core code was not changed.

File: docker-compose.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ services:
   covalent_dispatcher:
     build:
       context: .
-      dockerfile: refactor/dispatcher/Dockerfile
+      dockerfile: covalent_ispatcher/Dockerfile
```
Comment (venkatBala) [inline docker-compose.yml:25]: spelling fix `covalent_ispatcher` -> `covalent_dispatcher`

**Comment by scottwn** [comment]
I see a lot of commented code in this PR, is it possible to avoid that?

**Comment by kessler-frost** [comment]
Also need to build the webapp before trying to install it using `pip install dist/*tar.gz`.
Doing a `covalent status` also starts the services, `covalent stop` doesn't seem to stop the services.

**Comment by wjcunningham7** [comment]
> Also need to build the webapp before trying to install it using `pip install dist/*tar.gz`. Doing a `covalent status` also starts the services, `covalent stop` doesn't seem to stop the services.

yes you need to build the webapp first

**Pull request merged**